versions.txt

```diff
@@ -1,6 +1,6 @@
 # Format:
 # module:released-version:current-version
 
-bigtable-client-parent:2.12.1:2.12.2-SNAPSHOT
+bigtable-client-parent:2.12.1:2.13.0
 bigtable-hbase-replication:1.11.1:1.11.2-SNAPSHOT
```

Comment:
are all these released at the same time? We might need to bump these as well otherwise some artifacts may be referencing -SNAPSHOT versions

Reply:
fixed